cadc-util/src/main/java/ca/nrc/cadc/net/HttpTransfer.java

```diff
@@ -760,7 +760,7 @@ protected void checkErrors(URL url, HttpURLConnection conn)
         
         captureResponseHeaders(conn);
         
-        if (responseCode < 400) {
+        if (responseCode <= 100 && responseCode < 400) {
```

Comment:
In slack I wrote this in the slightly odd way (A <= B && B < C) because I can read it like math: A < B < C.

Here, you wrote it with the variable on the left of the operator instead of the right so it isn't "between" any more.

Reply:
ok.. hoping this is right now.